### PR TITLE
Add CONTRIBUTING.md

### LINKED ISSUE #10: Add CONTRIBUTING.md
Outline the tools we use. Outline that we use github flow. Outline anything else you have not known coming into this.

### PR DESCRIPTION
**Pull request by ctrlaltdev**
Fix #10

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+# Contributing to secops.space
+
+Before starting anything off, check in /issues if anything similar to what you want to do has been requested before.  
+If it never has, open a new feature request explaining in details your idea and wait for obbriggs or ojxhn to greenlight it.  
+Alternatively, you can touch base on IRC with fraq (obbriggs) or _0x6e756c6c (ojxhn) to expedite this process.
+
+## Branching Model and Git Flow
```
Comment (jxhn) [inline CONTRIBUTING.md]: This is all fantastic and I feel terrible for saying this but theres a subtle difference with gitflow and githubflow. The latter workswith github PRs a lot better. 

See with gitflow they mandate that master is sacred  and only gets merges from develop but we don't need that. The second and arguably bigger difference is that gitflow stipulates local merges then push to develop. This simply doesn't work with our PR + code review centric approach. 

So perhaps include this link https://guides.github.com/introduction/flow/

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+[  hotfix/bug ]       \__o__o                   /
+                       \     \                 /
+[ feature/new ]         \_____o_____o____o____o
+
+```
+See the actual [network of the repo](https://github.com/secops-space/secops.space/network).
+
+### Naming
+We prefix the name of the branch with its type: i.e. `feature/newfeature` or `hotfix/bugtofix`
+
```
Comment (jxhn) [inline CONTRIBUTING.md]: Maybe a sentence to let people know why.

It becomes 100000000x easier to glance at a branch and know whats what

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+## Linting and Prettifying
+
+We will soon use a linter to ensure a consistent way of writing code, and prevent some of your dirty weird habits to end up in prod.
+
+This will be finalized with #9
+
+## Dependencies
+
+Any addition of dependency must be thoroughly justified and evaluated.
```
Comment (jxhn) [inline CONTRIBUTING.md]: Because we like to keep things simple. This isn't the next myspace

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+This will be finalized with #9
+
+## Dependencies
+
+Any addition of dependency must be thoroughly justified and evaluated. 
+
+## Readability over fancyness / one liner / mind blowing shortcut
+
+We all enjoy witticism as a form of prowess, but far less when you come back to a piece of code months later and can't remember what it was for.
+
```
Comment (jxhn) [inline CONTRIBUTING.md]: Yaaaaaaaas queen to all of this

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
+# Contributing to secops.space
+
+Before starting anything off, check in /issues if anything similar to what you want to do has been requested before.  
+If it never has, open a new feature request explaining in details your idea and wait for obbriggs or ojxhn to greenlight it.  
+Alternatively, you can touch base on IRC with fraq (obbriggs) or _0x6e756c6c (ojxhn) to expedite this process.
```
Comment (jxhn) [inline CONTRIBUTING.md]: Might chat with nugget about this given the whole staff chat

File: CONTRIBUTING.md (hunk 3 above)
Comment (ctrlaltdev) [reply CONTRIBUTING.md]: It isn't? Damn...

File: CONTRIBUTING.md (hunk 5 above)
Comment (jxhn) [reply CONTRIBUTING.md]: @PaulGWebster @bbriggs Any changes you'd want here?

Comment (ctrlaltdev) [reply CONTRIBUTING.md]: WTF my mentions transformed to from '@...' to 'o...'

Comment (bbriggs) [reply CONTRIBUTING.md]: I was about to ask about that.

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+
+## Tech Stack
+
+The repo is based on [next.js](https://nextjs.org/) (server rendered react).
+
+Go to [README.md](README.md) for more info on how to set up your env.
+
+The website is hosted with [Zeit Now](https://zeit.co/now) and every commit to master is deployed automatically.  
+Also, every PR is build and deployed to make testing and code-review easy. So each PR is its own staging env.
```
Comment (bbriggs) [inline CONTRIBUTING.md]: s/build/built/

File: CONTRIBUTING.md (hunk 5 above)
Comment (ctrlaltdev) [reply CONTRIBUTING.md]: Fixed

**Pull request merged**